src/onegov/org/views/newsletter.py

```diff
@@ -82,15 +82,28 @@ def get_newsletter_form(
 
 def news_by_newsletter(
     newsletter: Newsletter,
-    request: 'OrgRequest'
+    request: 'OrgRequest',
+    visibility: str | tuple[str, ...] = 'public',
```

Comment:
Rename `visibility` to `access`. We should use the same name everywhere.

It also seems weird to pass in a list, just use one value and keep a list of access levels, if the index of the access of the News is lower than the index of the access provided, then it should be hidden.

I've used a similar idea for the `widest_access` in `onegov.org.utils`. You can reuse the `ORDERED_ACCESS` tuple and include everything below the index of the access given, i.e. `valid_accesses = ORDERED_ACCESS[ORDERED_ACCESS.index(access):]`